commands/trade.js

```diff
@@ -217,16 +217,16 @@ module.exports = function container (get, set, clear) {
               .replace(/\{\{symbol\}\}/g,  so.selector.normalized + ' - zenbot ' + require('../package.json').version)
             if (so.filename !== 'none') {
               var out_target
-              
+              var out_target_prefix = so.paper ? 'simulations/paper_result_' : 'stats/trade_result_'
```

Comment:
`stats/trade_result`?

Reply:
`trade_result_` was already the prefix of the file name. Stats, because I was checking the stats folder: https://github.com/DeviaVir/zenbot/tree/unstable/stats

The readme says *HTML Outputs from trades will be placed here!*
I've run several live modes and they were saved in the simulations folder, which doesn't make sense. They should be saved in the stats folder (even he readme says so, or did I misunderstood something?).

I added the variable `out_target_prefix` which is used a few lines below.

Reply:
@firepol Regarding your pullrequest question. If you want everything in a single commit. You can do the following:

First reset the HEAD (add a ^ for each commit you want to reset)  of your local branch:
```
git reset HEAD^
```

Then make your change and commit again. And force push to your remote branch, to overwrite the commtis in the pullrequest:

```
git push stats --force
```

Reply:
Thanks, I added `paper_result*` to the gitignore, reset my commit and forced pushed to it, github automatically updated it. Cool stuff.